router/realm.go

```diff
@@ -97,33 +107,10 @@ func NewRealm(uri wamp.URI, strictURI, anonymousAuth, allowDisclose bool) Realm
 	r.registerMetaProcedure(wamp.MetaProcRegCountCallees, r.regCountCallees)
 
 	go r.metaProcedureHandler()
-	return r
-}
-
-// AddAuthenticator registers the Authenticator for the specified method.
-func (r *realm) AddAuthenticator(method string, athr auth.Authenticator) {
-	r.actionChan <- func() {
-		r.authenticators[method] = athr
-	}
-	log.Printf("Added authenticator for method %s (realm=%v)", method, r.uri)
-}
-
-// DelAuthenticator deletes the Authenticator for the specified method.
-func (r *realm) DelAuthenticator(method string) {
-	r.actionChan <- func() {
-		delete(r.authenticators, method)
-	}
-	log.Printf("Deleted authenticator for method %s (realm=%v)", method, r.uri)
-}
-
-func (r *realm) SetAuthorizer(authorizer Authorizer) {
```

Comment:
How do we provide different Authorizers?  I did not see that as part of the RealmConfig.